wiki/en/Server-Install.md

```diff
@@ -0,0 +1,97 @@
+
+Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following guide assumes you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
+
+If you want to run a Server on a Raspberry Pi, you will need to [compile from source](https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md). See also this [guide for Raspberry Pi](/kb/2020/03/28/Server-Rpi.html) maintained by Jamulus user fredsiva.
+
+### Installation
+
+1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
+
+**Note**: You can upgrade your server by repeating the steps mentioned above.
+
+### Configuration
+
+You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+
+`sudo systemctl stop jamulus-headless`
+
+### Running an Unregistered Server
+
+It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
+
+### Port forwarding
+
+If you are running a Server on your home network, people from outside will not be able to see things inside it. So if you want to operate a Jamulus Server, you need to open a hole in your router's firewall to let Jamulus Clients connect to it.
+
+**The default port for the current version of Jamulus is UDP (not TCP) port 22124.**
+
+The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
+
+Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
+
+## Dynamic DNS and why you will probably need it
```

Comment:
The heading levels here are a total mess.

Level 3: `### Running an Unregistered Server`
Level 3: `### Port forwarding` (which applies regardless of server registration status)
Level 2: `## Dynamic DNS and why you will probably need it` (um...)
Level 4: `#### Running in Registered mode` ?!! This then is a sub-heading of Dynamic DNS?!  Should be same level as Unregistered.

Reply:
Agree. As I said in #707 the site is far from perfect. I think it should go into the direction of the install guides and Getting-Started page.

Reply:
This page is about installation.  Running as Registered, Unregistered and Directory servers should make no difference here.  Reference the "Running as" page for how to choose.  Explain that the topics covered apply to Registered, Unregistered and Directory servers.

Reply:
The section on installation is then fairly short and doesn't really warrant its own page.

Reply:
I think we should provide a setup guide comparable to https://www.facebook.com/notes/jamulus-online-musicianssingers-jamming/how-to-create-a-private-server-for-band-rehearsals/508642543044030/ or this for apache: https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-ubuntu-18-04